### PR TITLE
Color coding

### PR DESCRIPTION
**Pull request by AmjedXKhattab**

### CODE REVIEW

File: components/ColorCoding/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "English": "#1abc9c", // لون أخضر سماوي لفئة English
+  "Spanish": "#ecf0f1", // لون برتقالي لفئة Spanish
+  "French": "#95a5a6", // لون برتقالي غامق لفئة French
+  "Fitness Training": "#16a085", // لون أزرق لفئة Fitness Training
+  "Nutrition Consultation": "#27ae60", // لون أخضر فاتح لفئة Nutrition Consultation
+  "Mental Health Support": "#2980b9", // لون بنفسجي لفئة Mental Health Support
+  "Music Lessons": "#8e44ad", // لون أخضر سماوي لفئة Music Lessons
+  "Art Classes": "#d35400", // لون برتقالي لفئة Art Classes
+  "Photography Services": "#c0392b",
+  };
```
Comment (mbosselmann) [inline components/ColorCoding/index.js:17]: I think the formatting did not work? 🤔 And I think comment should be only in English to be understandable for your whole team. :)

Comment (mbosselmann) [inline components/ColorCoding/index.js:17]: I just tested some colors by adding entries and I think that some of the colors are too dark so the contrast between the text color and the background color is not high enough anymore? Please check. :)

File: components/ServiceProvider/index.js [hunk 2]
```diff
@@ -39,9 +40,12 @@ export default function ServiceProvider({
   serviceCards,
   setServiceCards,
 }) {
+
   const [showContactInfo, setShowContactInfo] = useState(false);
   const [editedCard, setEditedCard] = useState(null);
 
+  const backgroundColor = getColorForSubcategory(card.subcategory);
```
Comment (mbosselmann) [inline components/ServiceProvider/index.js:47]: Currently there is no difference between light and dark mode implemented. You should think about this too as between light and dark mode most colors are usually not the same. 🤔 Maybe integrating `subCategoryColors` into `lightTheme` and `darkTheme` can solve this (the `getColorForSubcategory` needs to be adjusted then)? :)

**Review by mbosselmann — COMMENTED** [review]
Hi Team ServiceCirle,
good job! 👍 I left some remarks for you to consider. :)

File: components/ColorCoding/index.js (hunk 1 above)
Comment (AmjedXKhattab) [reply components/ColorCoding/index.js:17]: Es tut mir leid, ich habe die Farben aus Chat gpt übernommen und vergessen, die Notizen zu löschen

Comment (AmjedXKhattab) [reply components/ColorCoding/index.js:17]: I searched for suitable colors but I did not find. Can you suggest good and suitable colors for the project?

Comment (mbosselmann) [reply components/ColorCoding/index.js:17]: @AmjedXKhattab Maybe you can find some help here for color palettes: https://coolors.co/ :)

**Review by doemser — APPROVED** [review]

**Pull request closed without merging**